.github/workflows/cmake.yml

```diff
@@ -139,6 +139,7 @@ jobs:
 
     - name: Build MegaGlest With Clang Compiler
       run: |
+        export CXXFLAGS=-std=c++11
```

Comment:
I don't know if this will help with 

```
/Users/runner/work/megaglest-source/megaglest-source/source/glest_game/game/game_settings.h:125:29: error: a space is required between consecutive right angle brackets (use '> >')
  vector<pair<string, uint32>> factionCRCList;
```

After I added it, the build broke with the error

```
2025-02-01T19:34:46.0273590Z /Applications/Xcode_15.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk/usr/include/gethostuuid.h:41:17: error: reference to 'uuid_t' is ambiguous
```